config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-main-upgrades.yaml

```diff
@@ -245,9 +245,9 @@ periodics:
       - name: KUBERNETES_VERSION_UPGRADE_FROM
         value: "stable-1.23"
       - name: KUBERNETES_VERSION_UPGRADE_TO
-        value: "ci/latest-1.24"
+        value: "stable-1.24"
       - name: ETCD_VERSION_UPGRADE_TO
-        value: "3.5.3-0"
```

Comment:
Should we keep 3.5.3 given that kubeadm didn't bump with v1.24 and we try to align to that usually? (not even on master)

If yes, let's make sure the other updated/added jobs are using 3.5.3 as well.